README.md

```diff
@@ -8,22 +8,48 @@ For full documentation visit [mkdocs.org](https://www.mkdocs.org).
 
 For full documentation visit https://squidfunk.github.io/mkdocs-material/.
 
-### Install
+### Quick Start
+
+> [!IMPORTANT]
+> to run the serve script, you need to first install [uv](https://docs.astral.sh/uv/getting-started/installation/).
+
+To quickly generate a view of the documenation
+
+Clone this repository on your PC/laptop, then view the documentation in a browser run `./serve`:
```

Comment:
It should be mentioned here that this workflow is based on forks. 
C2SM core team usually uses branches directly in this repo for PRs.